kernel/src/process.rs

```diff
@@ -178,12 +179,21 @@ impl ProcessId {
     }
 }
 
+/// A compressed form of an Application Identifer.
+#[derive(Clone, Copy, PartialEq, Eq)]
+pub struct ShortID {
```

Comment:
duplicate?

Reply:
I thought you suggested moving this from `process_checking` to here?

Reply:
Theres a shortid in appid.rs as well.

Reply:
Ah -- that's a vestigial file which should be removed.